modules/media/php/NDB_Menu_Filter_media.class.inc

```diff
@@ -72,32 +72,26 @@ class NDB_Menu_Filter_Media extends NDB_Menu_Filter
         $user =& User::singleton();
         $db   = Database::singleton();
 
-        $siteList  = Utility::getSiteList(false);
+        $siteList  = array();
         $visitList = Utility::getVisitList();
 
-        // Index sites using their names (used to filter react tables)
-        foreach ($siteList as $key => $site) {
-            unset($siteList[$key]);
-            $siteList[$site] = $site;
-        }
-
         // allow to view all sites data through filter
         if ($user->hasPermission('access_all_profiles')) {
-            // get the list of study sites - to be replaced by the Site object
-            if (is_array($siteList)) {
-                $siteList = $siteList;
+            $siteList = Utility::getSiteList(false);
+            // Index sites using their names (used to filter react tables)
+            foreach ($siteList as $key => $site) {
+                unset($siteList[$key]);
+                $siteList[$site] = $site;
             }
         } else {
             // allow only to view own site data
-            $siteList =array();
             $site_arr = $user->getData('CenterIDs');
-            foreach ($site_arr as $key => $val) {
-                $site[$key] = &Site::singleton($val);
-                if ($site[$key]->isStudySite()) {
-                    $siteList[$val] = $site[$key]->getCenterName();
+            foreach ($site_arr as $val) {
```

Comment:
Ok, last nitpick. Could you use `camelCase` for consistency? I know it's not consistent in Loris, but at least it stay consistent in `media` module.